.git-blame-ignore-revs

```diff
@@ -1,3 +1,14 @@
+# Commits in this file will be removed from GitHub blame results.
+#
+# To use this file locally, use:
+#   git blame --ignore-revs-file="path/to/.git-blame-ignore-revs" <files>
+#
+# If ignoring a pull request that was not squash merged, only the merge
+# commit needs to be put here. Child commits will be resolved from it.
```

Comment:
Don't know enough about how git blame works here (https://github.com/git/git/blob/f9b95943b68b6b8ca5a6072f50a08411c6449b55/blame.c#L2582 for the brave), but I'm a little suspicious here. I would guess that the merge commit shows up in blame if and only if 

- the file being blamed was in conflict
- there was a manual merge process: we didn't resolve by just checking out the file at one of the merge parents. (I.e.  the merge commit needed to create a blob object).

Happy to leave this as-is though.

Reply:
The merge conflict unintuitively does not show up in `git blame` - the actual commit does. However, only adding the merge commit to `.git-blame-ignore-revs` actually removed the changes from `git blame`.

I was mostly guessing that `git blame` follows children of merge commits based off the description of the [`--first-parent](https://git-scm.com/docs/git-blame#Documentation/git-blame.txt---first-parent) option.

One can try it themselves with:

merge commit:
```
git blame --ignore-rev 8b3d9b6b199abb87246f982d5db356f1966db925 tests/api/test_auth.py
```

vs.

```
git blame --ignore-rev a001038b92c4f0082cad60e5e87d0bd01374c9de tests/api/test_auth.py
```

and see that the latter makes no difference... unintuitively.